binary_search/Java/BinarySearch.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Arrays;
+
+/*
+    Binary search implemenetation that looks up index of a number in a collection.
+    If the number doesn't exist return -1.
+ */
+public class BinarySearch {
+
+    public static void main(String[] args) {
+	// write your code here
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpf4gqyt5s/binary_search/Java/BinarySearch.java
+++ b/tmp/tmpf4gqyt5s/binary_search/Java/BinarySearch.java
@@ -7,7 +7,7 @@
 public class BinarySearch {
 
     public static void main(String[] args) {
-	// write your code here
+    // write your code here
         int [] nums = {3,4,1,2,5};
         int key = 5;
         Arrays.sort(nums);
```